supervisor/docker/__init__.py

```diff
@@ -230,3 +230,20 @@ def repair(self) -> None:
             _LOGGER.debug("Networks prune: %s", output)
         except docker.errors.APIError as err:
             _LOGGER.warning("Error for networks prune: %s", err)
+
+    def check_denylist_images(self) -> bool:
+        """Return a boolean if the host has images in the denylist."""
+        denied_images = set()
+        for image in self.images.list():
+            for tag in image.tags:
+                image_name = tag.split(":")[0]
+                if (
+                    image_name in DOCKER_IMAGE_DENYLIST
+                    and image_name not in denied_images
+                ):
+                    _LOGGER.error(
+                        "Found image '%s', this is not supported, remove this from the host!",
+                        image_name,
+                    )
+                    denied_images.add(image_name)
+        return bool(denied_images)
```

Comment:
Prevent multible tags / error messages
```
if not denied_images:
   return False
_LOGGER.error(...)
return True
```

Reply:
Should we not make the user aware of all offending images they have so they can do something about it?
Currently, it logs 1 error pr image in the list

Reply:
`_LOGGER.error("We found such images: %s which is not acceptable....", denied_images)`